src/main/scala/com/github/flakm/InMemoryLdapServer.scala

```diff
@@ -97,10 +98,24 @@ object InMemoryLdapServer {
     val ds: InMemoryDirectoryServer = new InMemoryDirectoryServer(
       settings
     )
+    object Converter {
```

Comment:
Why have you created object here instead of just referencing `allFilesFromConfiguration` in the following code?

Reply:
Could you please elaborate? I gotta admit I'm not spending a lot of time in scala code bases (apart from own projects) maybe I'm missing something. I don't see anything wrong with it apart from indirection for a reader

Reply:
To encapsulate the responsibilities. but in that case you are right - it is too small to keep it.